src/consts.rs

```diff
@@ -0,0 +1,4 @@
+use qrcode::EcLevel;
+
+/// The error correction level to use for all QR code generation.
+pub const QR_EC_LEVEL: EcLevel = EcLevel::M;
```

Comment:
IIRC low error correction level was more suitable for the terminal when I tested it. The generated QR takes less space and reliable enough if we expect it to be scanned by high res camera. It might be true for the web page as well, though I haven't tested.

Reply:
Yeah sure we can use `EcLevel::L` if you believe it's more appropriate; I don't really have an opinion on this. If it comes with the immediate and tangible benefit of reducing screen footprint, why not. If it proves to be too low in the future it's always trivial to change it.